theme-plugin/src/main/resources/archetype-resources/pom.xml

```
@@ -36,7 +36,7 @@
   <properties>
     <changelist>999999-SNAPSHOT</changelist>
     <!-- https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/ -->
-    <jenkins.baseline>${jenkinsBaseline}</jenkins.baseline>
+    <jenkins.baseline>2.504</jenkins.baseline>
```

worth setting this to an old version to make sure it works after merging?


```suggestion
    <jenkins.baseline>2.492</jenkins.baseline>
```